src/screens/SavedEventListScreen/component.test.js

```diff
@@ -1,10 +1,10 @@
-// @flow
```

Comment:
If I keep flow in, it complains about my fake events. I tried copying the events from EventsList.test.js but complained about that as well. Ultimately decided it was adding value in this particular file.

Reply:
I suggest leaving this on. Otherwise, we face the issue in future when the data model gets updated that our tests are not testing with the correct data structures.

Reply:
the linter is complaining about it too :D I'll have another look :)

Reply:
.... I've just realised `EventList.test.js` also doesn't have flow in it, that'd be why it's failing the flow test!